src/Sulu/Bundle/AdminBundle/Resources/js/containers/FieldBlocks/tests/__snapshots__/FieldBlocks.test.js.snap

```diff
@@ -154,14 +154,14 @@ exports[`Render block with schema 1`] = `
             class="iconButtons"
           >
             <span
-              aria-label="su-trash-alt"
-              class="su-trash-alt clickable"
+              aria-label="su-more-circle"
```

Comment:
Any easy way to make this better accessible?

E.g.:

```html
<ul class="iconButtons"
   <li>
       <button ...>
           <span class="su-more-circle" aria-hidden="true"/>
           <span class="sr-only">More</span>
       </button>
   </li>
   <li>
       <button ...>
           <span class="su-collapse-vertical" aria-hidden="true"/>
           <span class="sr-only">More</span>
       </button>
   </li>
</ul>
```

Reply:
this is rendered using the `Icon` component:
https://github.com/sulu/sulu/blob/c298925a6815803b5f94ca80be3ffce1c939615f/src/Sulu/Bundle/AdminBundle/Resources/js/components/Block/Block.js#L100-L106

i can check if anything breaks if i use the `Button` component

Reply:
I would not use the button component just us a button tag ;)